package-lock.json

```diff
@@ -0,0 +1,11911 @@
+{
```

Comment:
Can you remove the `package-lock.json` and confirm the `yarn.lock` are up to date?